frontend/website/src/CryptoAppsSection.re

```diff
@@ -151,6 +152,16 @@ let make = _ => {
      .then((wallet) => Coda.sendTransaction(wallet, ...))
 </script>|}
           />
+          <div
+            className=Css.(
+              style([
+                width(`rem(13.75)),
+                height(`rem(0.)),
+                display(`none),
+                media(swapQuery, [display(`block)]),
+              ])
+            )
```

Comment:
What is this div for?

Reply:
It keeps the right hand text aligned (it's the same size as the svg below)